### DISCUSSION TITLE
[Projects Beta] create an issue automatically from project

### DISCUSSION THREAD

**Discussion by JulienGenoud** [community/community#15610] [Projects and Issues]
When i'm creating a new issue it's created as a draft,
can we link project to only one repo, in this way we create an issue directly from the project page without have to clic on "convert to an issue ?" 

<img width="220" alt="Capture d’écran 2022-04-26 à 10 06 22" src="https://user-images.githubusercontent.com/1808854/165252604-69b115f4-0547-4dd8-981b-c9b7490d34b8.png">

**Comment by JulienGenoud** [comment]
or maybe add a button here to convert a draft to your favorite project : 
<img width="643" alt="Capture d’écran 2022-04-28 à 11 19 35" src="https://user-images.githubusercontent.com/1808854/165721063-d7f97b07-1c94-4197-bdb0-c0be898f3472.png">

**Reply by itcarroll** [reply]
If you include/unhide "Repository" in the table view, it gives you a drop down menu that is essentially the button you want.

**Reply by haifengkao** [reply]
@itcarroll I cannot find the "Repository". Could you attach a screenshot?

**Reply by itcarroll** [reply]
In the menu below, clicking on "fields" brings up another menu with a checkbox for "Repository".

<img width="496" alt="Screen Shot 2022-12-01 at 9 20 57 AM" src="https://user-images.githubusercontent.com/3383837/205076750-282f18a7-2394-4537-9ea2-9ed259957357.png">

**Reply by JLScribani** [reply]
That doesn't appear to fix it. I have Repository hidden on my project view, but it still forces me to convert each draft to an issue and select the repo from a dropdown. I just want it to automatically convert drafts to issues.

**Reply by itcarroll** [reply]
@JLScribani Ah, I don't know how to do that. Projects and Repositories seem to be many-to-many, not one-to-one. I agree this could be a good feature.

**Reply by Paulmarkyoung0** [reply]
I would like to see this feature as well.

**Comment by mkruse83** [comment]
I don't know how long this feature already exists, but if you type now # into the box where you create the draft, it lets you select a repo and this stays active at least throughout the session. In my opinion this is good enough.

So this:

<img width="610" alt="image" src="https://github.com/community/community/assets/3939428/8fb288f3-00e0-4d90-be97-7f7a136947e7">

Turns into this:

<img width="610" alt="image" src="https://github.com/community/community/assets/3939428/2d5f2e5c-0f69-44e7-b397-e3036776e1d6">

**Reply by arobbins** [reply]
Thanks! Helped me.
